### PR TITLE
[Fix][Doc] fix doc tip style and add document format specification

### PR DESCRIPTION
**Pull request by bwcxyk**
- Added a new note section in the document format specification, including tips, notes, and cautions.
- Update the use of comments in Clickhouse, Hive, and LocalFile connector documentation.
- Added a note section in the Chinese version of the document format specification.

### Purpose of this pull request

This pull request fix doc tip style.

### Does this PR introduce _any_ user-facing change?

no

### How was this patch tested?

exists tests

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Review by Hisoka-X — COMMENTED** [review]
Thanks @bwcxyk ! Could you add a test case to check all docs in https://github.com/apache/seatunnel/blob/bd89d84fe655cd1b262e0cb1b83cbddc0d1a2f8d/seatunnel-ci-tools/src/test/java/org/apache/seatunnel/api/file/MarkdownTest.java#L36?

**Comment by bwcxyk** [comment]
> Thanks @bwcxyk ! Could you add a test case to check all docs in
> 
> https://github.com/apache/seatunnel/blob/bd89d84fe655cd1b262e0cb1b83cbddc0d1a2f8d/seatunnel-ci-tools/src/test/java/org/apache/seatunnel/api/file/MarkdownTest.java#L36
> 
> ?

I can't finish.

**Comment by bwcxyk** [comment]
Use [admonitions](https://docusaurus.io/docs/next/markdown-features/admonitions) to highlight important information.

**Comment by bwcxyk** [comment]
Use [admonitions](https://docusaurus.io/docs/next/markdown-features/admonitions) to highlight important information.

File: docs/en/contribution/docs-format-specification.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Docs Format Specification
+### Admonitions
```
Comment (Hisoka-X) [inline docs/en/contribution/docs-format-specification.md]:
```suggestion

## Admonitions

```

**Comment by Hisoka-X** [comment]
Please update https://github.com/apache/seatunnel/blob/bd89d84fe655cd1b262e0cb1b83cbddc0d1a2f8d/docs/sidebars.js#L229

File: docs/zh/contribution/docs-format-specification.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# 文档格式规范
+### 注释说明
```
Comment (Hisoka-X) [inline docs/zh/contribution/docs-format-specification.md]:
```suggestion

## 注释说明

```

**Review by Hisoka-X — APPROVED** [review]

**Review by hailin0 — APPROVED** [review]

**Review by davidzollo — APPROVED** [review]
+1
good job

**Pull request merged**